### ISSUE TITLE
SQLPro for MSSQL 2019.09.04 (10473) crashes frequently

### ISSUE THREAD

**Issue by William-Sheehan** [hankinsoft/SQLPro#319]
App crashes when scrolling in query window or upon opening .sql query file. Very frequent, (several times a day). Dump from "file open" crash attached. 
[2019-11-06 0637 SQLPro for MSSQL 94805 - Crash Dump.txt](https://github.com/hankinsoft/SQLPro/files/3814222/2019-11-06.0637.SQLPro.for.MSSQL.94805.-.Crash.Dump.txt)
This has always been a problem with this product, but previous versions autosaved my work and lost very little. As of recent releases, I loose all unsaved work when app crashes.

**Comment by hankinsoft** [comment]
Hi William,
First, sorry for the issue!

Can you try this build: https://d3fwkemdw8spx3.cloudfront.net/mssql/SQLProMSSQL.2019.53.app.zip. That should fix up the crashing issue you are seeing.

As for state recovery, what do you have set for the following two preferences:

<img width="657" alt="Screen Shot 2019-11-06 at 8 18 03 AM" src="https://user-images.githubusercontent.com/3160528/68297728-09db7000-006e-11ea-8256-2f1eccad29e1.png">

For clean recovery, the ideal state would be Autosaved checked and App Shutdown set to 'Save State'. (There were some bug fixes & improvements on state recovery between 09.04 and the build posted above).

Please give that a try and let me know if it helps out.

**Comment by William-Sheehan** [comment]
I'm actually on build 54 now and have not had any more crashes, (I have some other minor issues with the new build for which I'll open a new thread). 

Thanks for the tip on the settings, I did have "Autosave" checked, but I had "clean shutdown" selected for App Shutdown. I updated it to "Save State".  Hopefully it's a non-issue now as I don't have to worry about the crashing. 

Thanks, again for the help.
